io.openems.edge.battery.soltaro/src/io/openems/edge/battery/soltaro/cluster/versionb/Cluster.java

```diff
@@ -525,15 +389,22 @@ private void sleepSystem() {
 	}
 
 	private void resetSystem() {
-		// Write sleep and reset to all racks
+		// Write reset to all racks and the master
+		
+		IntegerWriteChannel resetMasterChannel = (IntegerWriteChannel) this.channel(ClusterChannelId.RESET);
+		try {
+			resetMasterChannel.setNextWriteValue(0x1);
+		} catch (OpenemsNamedException e) {
+			log.error("Error while trying to reset the master!");
```

Comment:
Ich nehme immer `this.logError(this.log, "Message...")` damit in der Konsole dann auch steht, von welcher Component-ID der Fehler kommt.